src/components/pages/Offers/Offer/Stocks/ActivationCodesUploadDialog/ActivationCodesCsvForm/ActivationCodesCsvForm.jsx

```diff
@@ -2,23 +2,43 @@ import PropTypes from 'prop-types'
 import React, { forwardRef, Fragment } from 'react'
 
 export const ActivationCodeCsvForm = forwardRef(function ActivationCodeCsvForm(
-  { isFileInputDisabled, submitThumbnail },
+  { isFileInputDisabled, submitThumbnail, errorMessage },
   ref
 ) {
   return (
     <Fragment>
-      <div className="activation-codes-upload-description">
-        <p>
-          {
-            'Pour les offres nécessitant une activation par code sur une plateforme extérieure, vous pouvez importer directement un fichier .csv.'
-          }
-        </p>
-        <p>
-          {
-            'Le stock disponible sera automatiquement mis à jour. Les jeunes auront accès à ce code dans leur espace réservation.'
-          }
-        </p>
-      </div>
+      {!errorMessage && (
+        <div className="activation-codes-upload-description">
+          <p>
+            {
+              'Pour les offres nécessitant une activation par code sur une plateforme extérieure, vous pouvez importer directement un fichier .csv.'
+            }
+          </p>
+          <p>
+            {
+              'Le stock disponible sera automatiquement mis à jour. Les jeunes auront accès à ce code dans leur espace réservation.'
+            }
+          </p>
+        </div>
+      )}
+      {errorMessage && (
+        <div className="activation-codes-upload-description">
+          <p>
+            {"Une erreur s'est produite lors de l'import de votre fichier "}
+            <b>
+              {`${ref.current.files[0].name}`}
+            </b>
+            {'.'}
+          </p>
+          <p>
+            {errorMessage}
+          </p>
+          <p>
+            {'Veuillez réessayer'}
```

Comment:
(typo / détail)

```suggestion
            {'Veuillez réessayer.'}
```

idem plus haut :)

Reply:
Là, pas d'excuse 👍